src/olgm/gm.js

```diff
@@ -405,11 +405,12 @@ export function createStyleInternal(style, mapIconOptions, opt_index) {
  * @param {number} index index for the label
  * @return {module:olgm/gm/MapLabel} map label
  */
-export function createLabel(textStyle, latLng, index) {
+export function createLabel(textStyle, latLng, index, pane) {
```

Comment:
missing `@param` above, and I think the pane parameter should be optional, i.e.: `{string=}` and the name of the 
parameter would be `opt_pane` (opt stands for optionnal)

Reply:
Added.